cirq-google/cirq_google/experimental/noise_models/noise_models.py

```diff
@@ -96,6 +114,21 @@ def noisy_moment(self, moment: cirq.Moment, system_qubits: Sequence[cirq.Qid]) -
                     )
             return moments
 
+    def _json_dict_(self) -> dict[str, object]:
+        return {
+            'depol_probs': tuple((q, p) for q, p in (self.depol_probs or {}).items()),
+            'bitflip_probs': tuple((q, p) for q, p in (self.bitflip_probs or {}).items()),
+            'decay_probs': tuple((q, p) for q, p in (self.decay_probs or {}).items()),
+        }
+
+    @classmethod
+    def _from_json_dict_(cls, depol_probs, bitflip_probs, decay_probs, **kwargs):
+        obj = cls.__new__(cls)
+        obj.depol_probs = dict(depol_probs)
+        obj.bitflip_probs = dict(bitflip_probs)
+        obj.decay_probs = dict(decay_probs)
```

Comment:
Please replace `cls.__new__` with standard object construction `cls(...)` here and in other files.  If `_json_dict_` above returns all `__init__` arguments, this method should be unnecessary and removed.